src/app/app.routing.ts

```diff
@@ -24,10 +36,15 @@ export const routes: Routes = [
         component: ForgotPasswordComponent,
         canActivate: [UnauthenticatedGuard]
     },
+
```

Comment:
ForgotPassword is added by Rajat, the extra space was an error, will correct it.